coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java

```diff
@@ -579,10 +579,17 @@ public boolean isOptional(int i) {
   /**
    * @return immutable copy of internal function registry
    */
-  public ImmutableMultimap<String, Function> getRegistry() {
+  public ImmutableMultimap<String, Function> getRegistryCopy() {
     return ImmutableMultimap.copyOf(FUNCTION_MAP);
   }
 
+  /**
+   * @return copy of internal function registry
+   */
+  public Multimap<String, Function> getRegistry() {
```

Comment:
needed this since TrinoToRelConverter needs the most up to date version of the map, the immutable map doesn't contain the new entries added in TrinoCalciteUDFMap